tests/test_data/simple_inputs/analysis.ini

```diff
@@ -4,6 +4,6 @@ name = simple_analysis
 [analysis1]
 name = single link redundancy test
 analysis = single_link_redundancy
-weighing = time
+weighing = distance
```

Comment:
yes, it's related to said issue, we were not using this test anywhere else (I checked) where it could have an impact. By using this simple test case I also dropped testing time in TC for the slow tests by 4 minutes (in parallel run).